TRADEMARK-COMMITTEE.md

```diff
@@ -58,12 +58,12 @@ subject to conformance and third-party licensing.
 
 ## Committee Meetings
 
-The committee meets as-needed. Meetings are held online.
+The trademark committee currently meets weekly, or as-needed. Meetings are open to public and held online.
 
 Given the private nature of some of these discussions (e.g. trade secrets, legal
-and compliance concerns, etc.) some meetings are held in private.
+and compliance concerns, etc.) some meetings may be held in private.
 
-Meeting notes are available to members of the
+[Meeting notes](https://docs.google.com/document/d/1ghGQl0O7TM8N_YQ33w6xKyQ2yUnOX0ACsOnlqoV_XgY/edit#) are available to members of the
```

Comment:
Same thing, include link to where folks can find recordings.

Reply:
I would be ok to reduce the links in the markdown document to reduce the chance to become stale (as @evankanderson mentioned above). Instead add 1 canonical link with a stable URL that then contains the links to more volatile links like calendar entries or other google docs.

E.g. having a single link to the meeting notes, and the meeting notes contain the links to calendar invite + recordings, and I would mention here that additional pointers like calendar entries or the location for the recordings can be found in the document linked, too.